db/migrate/20241204012338_create_local_businesses.rb

```diff
@@ -0,0 +1,18 @@
+class CreateLocalBusinesses < ActiveRecord::Migration[7.0]
+  def change
+    create_table :local_businesses do |t|
+      t.string :business_id
+      t.string :name
+      t.string :types
+      t.string :national_phone_number
+      t.string :fomatted_address
+      t.string :latitude
+      t.string :longitude
+      t.decimal :rating
+      t.string :buiness_status
+      t.json :json_data
```

Comment:
is there a difference between `json` and `jsonb`?

Reply:
you are right jsonb is better

Reply:
yeah its been helpful for where we've used it in RA